### PR TITLE
Update FreeBSD installation instructions

### PR DESCRIPTION
**Pull request by 0mp**
It requires splitting the FreeBSD and OpenBSD section into two sections
as those operating systems use different tools for package management.

Also, this patch updates only instructions in English.

### CODE REVIEW

File: locale/en/download/package-manager.md
Comment (vsemozhetbyt) [inline locale/en/download/package-manager.md]: Nit: missing period or colon at the end)

File: locale/en/download/package-manager.md
Comment (vsemozhetbyt) [inline locale/en/download/package-manager.md]: Nit: should go after `[nvm](#nvm)` item)

File: locale/uk/download/package-manager.md
Comment (vsemozhetbyt) [inline locale/uk/download/package-manager.md]: `[OpenBSD)(#openbsd)` -> `[OpenBSD](#openbsd)`?

File: locale/uk/download/package-manager.md
Comment (vsemozhetbyt) [inline locale/uk/download/package-manager.md]: Nit: missing period or colon at the end)

**Review by vsemozhetbyt — COMMENTED** [review]
Thank you!

**Comment by vsemozhetbyt** [comment]
cc @nodejs/platform-freebsd

**Comment by No9** [comment]
LGTM :+1: On the FreeBSD pieces

**Comment by emaste** [comment]
Overall LGTM with a couple of comments/questions; thanks for the update.

The GitHub review UI seems to be nonfunctional for me at the moment so apologies for not commenting inline.

```
+The most recent release of Node.js is always available
```
Perhaps we should temper this slightly by removing `always` or a similar change, because there may be a small amount of latency between a release happening and ports being updated?

Markdown parser accepts both `sh` and `bash`?

**Comment by bradleythughes** [comment]
@emaste 
>Perhaps we should temper this slightly by removing always or a similar change, because there may be a small amount of latency between a release happening and ports being updated?

I second this. I am the maintainer of www/node, and in general I try to keep the port up-to-date with the latest release, but there will always be a delay because I tend to discover new releases by accident, most often several days after the fact. WTB an announcement mailing list or other notification mechanism that can make sure that package maintainers do not have to resort to polling the blog or Github release page for new releases. 🙂

**Comment by 0mp** [comment]
First of all, I'm sorry for all those typos.

I've added the changes you've requested. Tell me if you'd like me to rebase before this PR gets merged.

**Comment by vsemozhetbyt** [comment]
@0mp No need to be sorry, our doc PRs contain some nits most of the time)

**Comment by vsemozhetbyt** [comment]
As for the [comment](https://github.com/nodejs/nodejs.org/pull/1658#issuecomment-390399536): cc @nodejs/release.

**Review by vsemozhetbyt — APPROVED** [review]
Doc format LGTM)

**Review by lpinca — APPROVED** [review]

**Review by daxlab — APPROVED** [review]

**Review by WaleedAshraf — APPROVED** [review]
LGTM

**Pull request merged**
